crate_universe/Cargo.toml

```diff
@@ -14,13 +14,13 @@ edition = "2018"
 # Several of these dependencies need to be compatible with the version cargo-raze is using.
 
 anyhow = "1"
-cargo-raze = "0.9"
-cargo_metadata = "0.12.1"
-cfg-expr = "0.4.1"
-env_logger = "0.7"
+cargo-raze = { git = "https://github.com/UebelAndre/cargo-raze.git",  branch = "universe", default-features = false }
```

Comment:
Can you change this to use a `rev` reference pointing at a specific sha, rather than a branch name? Ideally we don't want this to float if you push new things to the branch :)

Reply:
updated